backend/tests/v5/unit/middleware/dataConverter/inputs/teamspaces/projects/models/commons/models.test.js

```diff
@@ -0,0 +1,294 @@
+/**
+ *  Copyright (C) 2021 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+const { src } = require('../../../../../../../../helper/path');
+
+jest.mock('../../../../../../../../../../src/v5/utils/responder');
+const Responder = require(`${src}/utils/responder`);
+const { createResponseCode, templates } = require(`${src}/utils/responseCodes`);
+
+const Models = require(`${src}/middleware/dataConverter/inputs/teamspaces/projects/models/commons/models`);
+
+// Mock respond function to just return the resCode
+Responder.respond.mockImplementation((req, res, errCode) => errCode);
+
+const testValidateAddModelData = () => {
+	describe('Validate model data', () => {
+		test('next() should be with a valid request body', async () => {
```

Comment:
use described.each().

There's too much copy and paste and it's difficult to add new tests.